purchase_open_qty/models/purchase_order.py

```diff
@@ -10,9 +10,9 @@
 class PurchaseOrderLine(models.Model):
     _inherit = 'purchase.order.line'
 
-    @api.depends('invoice_lines.invoice_id.state', 'product_qty')
-    def _compute_qty_invoiced(self):
-        super(PurchaseOrderLine, self)._compute_qty_invoiced()
+    @api.depends('invoice_lines.invoice_id.state', 'product_qty',
```

Comment:
Now `invoice_lines.invoice_id.state` is not required since that you have `qty_invoiced`